packages/components/src/components/breadcrumb/breadcrumb-item-label.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, h, Prop, Element } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-breadcrumb-item-label',
+  styleUrl: 'breadcrumb-item-label.scss',
+  shadow: true,
+})
+export class BreadcrumbItemLabel {
+  @Prop() icon: string;
+  @Prop() url: string;
+  @Prop() target: string = "_self"
+  @Element() el;
+
+
+  render() {
+    return (
+      <a href={this.url} target={this.target} class="breadcrumb-item-label-container">
+        <ifx-icon icon={this.icon}></ifx-icon>
```

Comment:
You might want to consider adding error handling if the url or icon props are not provided

Reply:
But why it's not a requirement to have an icon or a url. Those props are optional. Why would there be an error for failing to do something optional?

Reply:
error handling does not mean "throw an error if null".

What we want to do here is: We want to have "javascript:void(0)" instead of the href if it is null. We don't want to show the icon if it is null

Reply:
That's why I have this:
.breadcrumb-item-label-container {
  & ifx-icon:empty { 
    display: none;
   }
  }
  
  If the icon is not rendered, either because the name was invalid, or because it does not exist, the component itself will not be rendered.